### PR TITLE
RHS GREF CDF PvP loadout rework

### PR DESCRIPTION
**Pull request by ComboTombo**
## What type of PR is this.
1. [ ] Bug
2. [x] Enhancement

### What have you changed and why?
Information: I am currently working on a full rework on the PvP loadouts. This rework will change a majority of the loadouts, everything from weaponry, gear, and equipment to ammunition, medical items, and misc. items. If you want to get a bit more details on the loadout changes, you can visit my repo and check the issues labeled "documentation" (for lack of a better word).
    

### Please specify which Issue this PR Resolves.
N/A

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the Mission in Singleplayer?
2. [x] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

********************************************************
Notes: **As per Meerkat's request, I've created this PR and moved my CDF changes from #979 to this PR.**

### CODE REVIEW

File: A3-Antistasi/Templates/Loadouts/rhs_gref_machineGunner.sqf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		]
+		+ _aceFlashlight
+		+ _aceM84
+	],
+
+	[																							//Backpack
+		[																			          //Backpack
+    []																				          //Inventory
+    ]
+	],
```
Comment (CalebSerafin) [inline A3-Antistasi/Templates/Loadouts/rhs_gref_machineGunner.sqf:53]: The formatting needs a touch up; Tabs are mixed with spaces.
```Suggestion    
    [ //Backpack
        [                                                                                   //Backpack
            [] //Inventory
        ]
    ],
```

File: A3-Antistasi/Templates/Loadouts/rhs_gref_marksman.sqf [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+		+ _aceFlashlight
+		+ _aceM84
+		+ _aceRangecard
+	],
+
+	[																							//Backpack
+		[																			          //Backpack
+    []																				          //Inventory
+    ]
+	],
```
Comment (CalebSerafin) [inline A3-Antistasi/Templates/Loadouts/rhs_gref_marksman.sqf:54]: Same issue as machine gunner:
```Suggestion
    [ //Backpack
        [                                                                                   //Backpack
            [] //Inventory
        ]
    ],
```

File: A3-Antistasi/Templates/Loadouts/rhs_gref_teamLeader.sqf [hunk 3]
```diff
@@ -1,66 +1,62 @@
...
-		"B_LegStrapBag_Black_F",							//Backpack
-		[//Inventory
-			["RHS_Mag_30Rnd_556x45_M855_PMag",3,30],
-			["16Rnd_9x21_Mag",2,17]
-		]
+	[																							//Backpack
+		[																			          //Backpack
+    []																				          //Inventory
+    ]
 	],
```
Comment (CalebSerafin) [inline A3-Antistasi/Templates/Loadouts/rhs_gref_teamLeader.sqf:53]: Formatting
```suggestion
    [ //Backpack
        [                                                                                   //Backpack
            [] //Inventory
        ]
    ],
```

**Comment by Lazejun** [comment]
**Vests:**
"rhsgref_6b23_ttsko_mountain_nco" or "rhsgref_6b23_ttsko_mountain_officer" either for TL.

"rhsgref_6b23_ttsko_mountain_sniper" Marksman.

"rhsgref_6b23_ttsko_mountain" AT and Machinegunner.

"rhsgref_6b23_ttsko_mountain_rifleman" Looks Terrible.



**Headgear:**
"rhsgref_ssh68_ttsko_mountain" Matches the Camo.


"rhsgref_6b27m_ttsko_mountain" Doesent Matches the Camo.

**Comment by ComboTombo** [comment]
> Solid additions. Some minor suggestions for formatting:
> 
> 1. We are phasing out tab indentation for spaces. So it would be nice replacing the tabs with spaces.

Sure!

> 2. The comments are out of line; this is most likely due to mismatching rendering of tabs.

Yeah. Atom uses 2-space tabs and GitHub uses 8-space tab. To view the proper format you'll need to place "?ts=2" at the end of the URL (only works when viewing the file itself).

> 3. See the 3 comments.

I assume you mean the other 3 comments. Sure.

> 4. The Headgear and face mask has an extra indentation level. I know this already existed in the templates so it's not your error, however, you can correct it if you want.

Didn't even notice, good catch!

> 5. I assume that there are zero errors from classname typo's as that will take a long time for me to verify.

Not that I could find any. Everything seems to be working fine.

> 
> Otherwise, well done! 🥳

Thank you!

**Comment by Bob-Murphy** [comment]
@SonorousMeerkat please give your OK once this is finished and five the tag "ready for merge" when done.

**Pull request merged**